src/Core/Framework/Adapter/Twig/SecurityExtension.php

```diff
@@ -54,8 +54,14 @@ public function map(?iterable $array, string|callable|\Closure $function): ?arra
 
         $result = [];
         foreach ($array as $key => $value) {
-            // @phpstan-ignore-next-line
-            $result[$key] = $function($value);
+            if (\is_string($function)) {
+                // Custom functions
+                // @phpstan-ignore-next-line
```

Comment:
would you mind changing it to this? 
```suggestion
                // @phpstan-ignore specific.identifier (comment why we need to ignore it)
```

also see https://phpstan.org/user-guide/ignoring-errors

So there is more context, why we have to ignore a PHPStan error and additionally only this specific error is ignored and not the whole line

Reply:
I dont't know 🤷 it was there before, so I kept it. 🙈

Reply:
Changed as discussed in our call. 😄